### PR TITLE
Fix arm pandoc test

### PR DESCRIPTION
**Pull request by mathbunnyru**

### CODE REVIEW

File: base-notebook/Dockerfile [hunk 1]
```diff
@@ -28,13 +28,14 @@ ENV DEBIAN_FRONTEND noninteractive
...
     apt-get install --yes --no-install-recommends \
-    tini \
-    wget \
     ca-certificates \
-    sudo \
-    locales \
     fonts-liberation \
-    run-one && \
+    locales \
+    pandoc \
```
Comment (mathbunnyru) [inline base-notebook/Dockerfile:34]: Sorted, and added this line.
Pandoc is not installed by default in arm images.

File: test/test_packages.py [hunk 2]
```diff
@@ -62,7 +62,6 @@
 
 # List of packages that cannot be tested in a standard way
 EXCLUDED_PACKAGES = [
-    "tini",
```
Comment (mathbunnyru) [inline test/test_packages.py:65]: This is system package, it won't be tested at all, so we don't need to exclude it.

**Comment by mathbunnyru** [comment]
`jupyter/base-notebook` will have only one failing test, i hope it's gonna be fixed in the near future.
And, I will try to fix other arm issues as well.

File: base-notebook/Dockerfile (hunk 1 above)
Comment (consideRatio) [reply base-notebook/Dockerfile:34]: Can you make inline note about that?

This is knowledge that can be useful to retain like an inline comment i think

Comment (mathbunnyru) [reply base-notebook/Dockerfile:33]: Done, please, take a look

**Review by consideRatio — APPROVED** [review]

**Pull request merged**